src/sch/eager.jl

```diff
@@ -3,9 +3,43 @@ const EAGER_THUNK_CHAN = Channel(typemax(Int))
 const EAGER_ID_MAP = Dict{UInt64,Int}()
 const EAGER_CONTEXT = Ref{Context}()
 const EAGER_STATE = Ref{ComputeState}()
+const EAGER_SCH_OPTS = Ref{SchedulerOptions}(SchedulerOptions(;allow_errors=true))
 
 eager_context() = isassigned(EAGER_CONTEXT) ? EAGER_CONTEXT[] : nothing
 
+"""
+    global_eager_scheduler_options(; kwargs...)
+
+Set / retrieve the global eager scheduler options.
+If no `kwargs` are passed, the current `SchedulerOptions` is returned,
+otherwise the global eager scheduler options are set to
+`SchedulerOptions(;kwargs...)` and returned.
+# Keywords
+- `kwargs...`: SchedulerOptions keywords to be set
+# Returns
+- `SchedulerOptions`: The global options used for eager scheduling.
+"""
+function global_eager_scheduler_options(; kwargs...)
+    if !isassigned(EAGER_SCH_OPTS) || !isempty(kwargs)
```

Comment:
The `isassigned(EAGER_SCH_OPTS)` isn't really necessary since you already set it at module load time, right?

Reply:
Note that you also need to update the docstrings.